horovod/common/operations.cc

```diff
@@ -768,6 +768,23 @@ bool CheckForStalledTensors(HorovodGlobalState& state, MPIContext& ctx) {
   return should_shut_down;
 }
 
+// Check for cached tensors that have been pending for a long time.
+void CheckForStalledCachedTensors(HorovodGlobalState& state,
```

Comment:
Rename to `InvalidateStalledCachedTensors`?

Reply:
Done!